npm/tailcall/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@tailcallhq/tailcall",
+  "version": "0.13.0",
+  "type": "module",
+  "homepage": "https://tailcall.run",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tailcallhq/tailcall.git"
+  },
+  "scripts": {
+    "gen": "node scripts/generate-packages.js"
+  },
+  "author": "TailcallHQ",
+  "license": "Apache-2.0",
+  "bugs": "https://github.com/tailcallhq/tailcall/issues",
+  "description": "A high-performance GraphQL proxy, optimized for cloud-native environments, covering edge, middle, and service layers.",
+  "keywords": [
+    "GraphQL",
+    "Rust",
+    "microservices"
+  ],
+  "engines": {
+    "node": ">=14.*"
+  }
+}
```

Comment:
This should be a private package. 
set `private:true`.

Add workspaces for each supported platform.